command.go

```diff
@@ -41,14 +41,16 @@ func (c command) exec(debug bool, verbose bool) (int, string, string) {
 		}
 	}
 
+	command := c.Cmd + " " + strings.Join(args, " ")
+
 	if debug {
 		log.Println("INFO: " + c.Description)
 	}
 	if verbose {
-		log.Println("VERBOSE: " + c.Cmd + " " + strings.Join(args, " "))
+		log.Println("VERBOSE: " + command)
 	}
 
-	cmd := exec.Command(c.Cmd, args...)
+	cmd := exec.Command("sh", "-c", command)
```

Comment:
why is this needed? what error where you getting when calling the command directly?

Reply:
the override flags were not working because regular exec.command ignores quotes in args.
so this existing override  was never passed to tiller 
[]string{"--override", "'spec.template.spec.containers[0].command'='{/tiller,--storage=secret}'"}
and my overrides were not working as well

Reply:
Ah, I see. In that case the solution should be to include the quotes in the string, something like:
```go
[]string{"--override", `"'spec.template.spec.containers[0].command'='{/tiller,--storage=secret}'"`}
```
What do you think?

Reply:
Including quotes did not work, for example the command you posted was not being processed, instead of globally changing it to "sh -c" though I changed just that command to run as "sh -c" (got that idea from the command_test.go). Otherwise "sh -c" was breaking all the tests for directories with white space (that we may want to consider not supporting but doesn't matter too much to me). I updated to add documentation as well.

Reply:
the line I wrote was just an example, how if the final command supposed to look like, with quotes? There must be a solution, the problem with using `sh -c` is that it won't work for windows users...

Reply:
Hmm not sure, will need to think about it, try it out though maybe I was incorrect but the 'spec.template.spec.containers[0].command'='{/tiller,--storage=secret}' never ends up getting set on your tiller in this current version

Reply:
I think go isn't honoring the single quotes around the different pieces of the --override and helm requires those otherwise it ignores the override